selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/CreateFactoryTest.java

```diff
@@ -196,9 +200,9 @@ private void createWorkspaceWithProject(String workspaceName) {
     workspaces.clickOnAddWorkspaceBtn();
     newWorkspace.waitToolbar();
     loader.waitOnClosed();
+    newWorkspace.clickOnAllStacksTab();
```

Comment:
please, comment on why we can't rely on Quick Start stack list.

Reply:
The **Java** stack, which we use for creating test workspaces, not exist **Quick Start** stack list on **OSIO**.